modules/schema/src/main/java/org/apache/ignite/internal/schema/DefaultValueGenerator.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.schema;
+
+import java.lang.reflect.Method;
+import java.util.UUID;
+
+/**
+ * Enumeration of all supported value generators that could be used as a default value provider
+ * (i.e. could be specified as a default in column definition).
+ */
+public enum DefaultValueGenerator {
+    /** Generator that generates random UUID string. */
+    GEN_RANDOM_UUID("genRandomUuid");
+
+    private final Method method;
+
+    DefaultValueGenerator(String methodName) {
+        try {
+            this.method = DefaultValueGenerator.class.getMethod(methodName);
```

Comment:
Can we use MethodHandle here?
From a performance standpoint, the MethodHandles API can be much faster than the Reflection API since the access checks are made at creation time rather than at execution time.

I think. In case of user-defined function, Reflection API looks preferable due to security reasons.
For standard functions like 'randomUUID' MethodHandle API is acceptable
WDYT?

Reply:
It could be different from one JDK to another JDK

Reply:
The results may depends on security provider. Reflection method will check access rules on each call.

Why don't you use Blackhole to consume the results?
```suggestion
            @Benchmark
    public void methodHandler(Blackhole bh) {
        bh.consume(DefaultValueGenerator.GEN_RANDOM_UUID.nextMethodHandle());
    }
```

Reply:
> The results may depends on security provider. Reflection method will check access rules on each call.

make sense. Fixed

> Why don't you use Blackhole to consume the results?

Because JMH will use blackhole implicitly for returned values